dialogflow-app.js

```diff
@@ -962,10 +965,10 @@ class DialogflowApp extends AssistantApp {
       if (textToSpeech.speech) {
         // Convert SimpleResponse to RichResponse
         textToSpeech = this.buildRichResponse().addSimpleResponse(textToSpeech);
-      } else if (!(textToSpeech.items &&
+      } else if (!((textToSpeech.items &&
         textToSpeech.items[0] &&
-        textToSpeech.items[0].simpleResponse)) {
-        error('Invalid RichResponse. First item must be SimpleResponse');
+        textToSpeech.items[0].simpleResponse) || textToSpeech.event)) {
+        error('Invalid RichResponse. First item must be simpleResponse or contains an event');
```

Comment:
"First item must be a SimpleResponse or contain an event"

Reply:
But I don't know if we want the first item to be an event.

Reply:
I think a rephrase should clarify. Indeed the items and events go separately. Items don't include events, but the response may have them.